### PR TITLE
feat: add and fix translation to installation caption

### PR DESCRIPTION
**Pull request by Michael-Liendo**

### CODE REVIEW

**Comment by Michael-Liendo** [comment]
How will the code be maintained, for example the outputs in console, in English or Spanish?

**Comment by deavid** [comment]
> How will the code be maintained, for example the outputs in console, in English or Spanish?

I prefer to leave the console outputs verbatim (English). I don't see much advantage on translating those and it takes a lot of effort in maintaining the translation.

File: po/es.po [hunk 1]
```diff
@@ -1979,9 +1978,16 @@ msgid ""
 "analyzer.github.io/manual.html#vimneovim), and many others. There is also a "
 "different IDE available called [RustRover](https://www.jetbrains.com/rust/)."
 msgstr ""
+"Después de instalar Rust, tu puedes configurar tu editor o IDE para trabajar "
```
Comment (deavid) [inline po/es.po:1981]:
```suggestion
"Después de instalar Rust, deberías configurar tu editor o IDE para trabajar "
```

File: po/es.po [hunk 2]
```diff
@@ -2201,7 +2210,7 @@ msgstr "Los bloques de código de este curso son totalmente interactivos:"
 
 #: src/cargo/code-samples.md src/cargo/running-locally.md
 msgid "\"Edit me!\""
-msgstr ""
+msgstr "\"Edítame\""
```
Comment (deavid) [inline po/es.po:2213]:
```suggestion
msgstr "\"Edítame!\""
```

File: po/es.po [hunk 3]
```diff
@@ -2252,16 +2260,18 @@ msgid ""
...
+"(https://doc.rust-lang.org/book/ch01-01-installation.html). De este modo, "
+"obtendrás un  `rustc` y un `cargo` que funcionen. En el momento de escribir "
+"esto, la última versión estable de Rust tiene estos números de versión:"
 
 #: src/cargo/running-locally.md
 msgid ""
 "You can use any later version too since Rust maintains backwards "
 "compatibility."
 msgstr ""
+"Tu puedes usar cualquier versión de Rust, ya que Rust mantienen "
```
Comment (deavid) [inline po/es.po:2273]:
```suggestion
"Puedes usar cualquier versión de Rust, ya que Rust mantienen "
```

**Review by deavid — CHANGES_REQUESTED** [review]
All good, just a few suggestions

**Comment by Michael-Liendo** [comment]
> I prefer to leave the console outputs verbatim (English). I don't see much advantage on translating those and it takes a lot of effort in maintaining the translation.

Okey!

**Review by deavid — APPROVED** [review]

**Pull request merged**